src/yoroi-wallets/cardano/api/api.ts

```diff
@@ -233,8 +233,9 @@ function parseNFTs(value: unknown, storageUrl: string): YoroiNft[] {
     const nftMetadata = nftAsset.metadata?.[policyId]?.[assetName]
 
     if (!nftMetadata || !nftMetadata.image) {
```

Comment:
no metadata -> default its fine, but if it is missing only `.image` I'd merge `{...default, ...incomingMetadata}`